### PR TITLE
Fix style and create new session for talks 🗣️

### LINKED ISSUE #97: Fix <a> color when using dark mode
When we are using the dark mode. the link colors aren't so legible. 

image

I would like to fix this issue since is an up-for-grabs and I am new contributing here.

The main goal with this issue is only to foster communication.

@aaronamm can we pair together on this?

### PR DESCRIPTION
**Pull request by kidchenko**
On this PR we fix #97 and add a section for .NET Thailand Talks. We can use that page to link and keep the documentation, slides, and video of the meetups, conferences,s and other events.

What this PR solves?

We change the color of the link using CSS in JS. I also added hover.

I would like to use a variable or a color from the theme? @aaronamm are you aware of how to do it?

Thanks :)

Let me know what do you think.

Current: 

image

image

image

### CODE REVIEW

**Comment by kidchenko** [comment]
image

**Review by aaronamm — APPROVED** [review]
Thank you for your PR. I will merged it.

File: README.md [hunk 1]
```diff
@@ -10,14 +10,20 @@
 See [CONTRIBUTING](./CONTRIBUTING.md).
 
 ## How to run this project locally
-- Install `Yarn` and `Gatsby` as a global tool.
```
Comment (aaronamm) [inline README.md]: Good caught

**Comment by aaronamm** [comment]
@kidchenko there is a conflict in README.md. Therefore, I rebased your branch with master and force push.

Thank you so much for your contribution.

**Pull request merged**